.github/workflows/build-ci.yml

```diff
@@ -39,40 +42,48 @@ jobs:
                     sleep 1
                     done
                     sudo docker exec --tty mongodb $MONGOSH_BIN 127.0.0.1:27017 --eval "rs.initiate({\"_id\":\"rs\",\"members\":[{\"_id\":0,\"host\":\"127.0.0.1:27017\" }]})"
+
             -   name: Show MongoDB server status
                 run: |
                     if [ "${{ matrix.mongodb }}" = "4.4" ]; then MONGOSH_BIN="mongo"; else MONGOSH_BIN="mongosh"; fi
                     docker exec --tty mongodb $MONGOSH_BIN 127.0.0.1:27017 --eval "db.runCommand({ serverStatus: 1 })"
-            -   name: "Installing php"
+
+            -   name: Installing php
```

Comment:
Quoting strings in Yaml is a good practice to avoid unexpected behaviors like `no` being converted to `false`. So I'm not in favor of removing this quotes everywhere.